rustls/tests/ech.rs

```diff
@@ -0,0 +1,100 @@
+mod ech_configs {
```

Comment:
I was thinking this file would have additional ECH related tests in the future and it might be nice to group the config related ones in their own module.

I don't feel strongly though, happy to flatten the namespace.

Reply:
I would flatten for now, can reintroduce later if we think it makes sense.